sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -50,44 +56,98 @@ def load(cls, connection_string=None, endpoint=None, credential=None, **kwargs):
 
         key_vault_options = kwargs.pop("key_vault_options", None)
 
-        provider.__buildprovider(connection_string, endpoint, credential, key_vault_options)
+        if connection_string and endpoints:
+            raise AttributeError("Both connection_string and endpoints are set. Only one of these should be set.")
+        if endpoint and endpoints:
+            raise AttributeError("Both endpoint and endpoints are set. Only one of these should be set.")
+
+        # TODO: Currently Connection Strings are not supported for Geo Replication
+        if endpoints:
+            for endpoint_geo in endpoints:
+                provider._clients.append([endpoint_geo, provider.__build_provider(None, endpoint_geo, credential,
+                    key_vault_options)])
+        else:
+            store_endpoint = endpoint if endpoint is not None else provider.__parse_connection_string(connection_string)
+            provider._clients.append([store_endpoint, provider.__build_provider(connection_string, endpoint,
+                credential, key_vault_options)])
 
         selects = kwargs.pop("selects", {SettingSelector("*", "\0")})
```

Comment:
Not related to this pr but it seems the keyword for "selects" has been specified as "selectors" on line 48. Correct me if I'm wrong about this.

Reply:
Good catch. I have a different PR to update few small things. It will go into that.